custom_components/tuya_local/devices/orbegozo_rrw_emisor_termico.yaml

```diff
@@ -0,0 +1,96 @@
+name: Heater
+products:
+  - id: bf49669b1da42ca7f34faa
+    manufacturer: Orbegozo
+    model: RRW Emisor Termico
+entities:
+  - entity: climate
+    translation_only_key: heater
+    dps:
+      - id: 1
+        name: hvac_mode
+        type: boolean
+        mapping:
+          - dps_val: true
+            value: "heat"
+          - dps_val: false
+            value: "off"
+      - id: 2
+        name: temperature
+        type: integer
+        unit: C
+        range:
+          min: 7
+          max: 35
+        mapping:
+          - constraint: preset_mode
+            conditions:
+              - dps_val: ANTIFROST
+                range:
+                  min: 7
+                  max: 28
+          - constraint: preset_mode
+            conditions:
+              - dps_val: ECONOMY
+                range:
+                  min: 4
+                  max: 32
+      - id: 3
+        name: current_temperature
+        type: integer
+      - id: 4
+        name: preset_mode
+        type: string
+        mapping:
+          - dps_val: COMFORT
+            value: Comfort
```

Comment:
Use values that match standard HA presets where possible  (all lower case, as they are translation keys): `comfort`, `eco`, `away`. 

Program does not have a direct equivalent, but I think `home` is often used for such settings.